src/Nox.Types/Types/CultureCode/CultureCode.cs

```diff
@@ -7,8 +7,19 @@ namespace Nox.Types;
 /// </summary>
 public class CultureCode : ValueObject<string, CultureCode>
```

Comment:
The comments here are a bit redundant, since the names of the constants are already descriptive.